e2e/commands/drug-order-operations.ts

```diff
@@ -0,0 +1,43 @@
+import { type APIRequestContext, expect } from '@playwright/test';
+import { type Order } from '@openmrs/esm-patient-common-lib';
+import dayjs from 'dayjs';
+
+export const generateRandomDrugOrder = async (
+  api: APIRequestContext,
+  patientId: string,
+  encounterUuid: string,
+  providerUuid: string,
+): Promise<Order> => {
+  const order = await api.post('order', {
+    data: {
+      orderType: '131168f4-15f5-102d-96e4-000c29c2a5d7',
+      type: 'drugorder',
+      action: 'new',
+      drug: '09e58895-e7f0-4649-b7c0-e665c5c08e93',
+      urgency: 'ROUTINE',
+      dateActivated: dayjs().format(),
```

Comment:
I think rather than just passing in the UUID of the encounter, we should pass in the created encounter and ensure the `dateActivated` is on or after the Encounter datetime.